src/Resque/Job.php

```diff
@@ -803,4 +803,17 @@ public static function cleanup(array $queues = array('*'))
 
         return $cleaned;
     }
+
+    /**
+     * Update the Delayed Job Status to STATUS_CANCELLED manually
+     *
+     * @param string $JobId
+     */
+
+    public static function cancelJob($JobId) {
+        //const STATUS_CANCELLED = 5;
+        if($JobId != '') {
+            Redis::instance()->hset(self::redisKey($JobId), 'status', 5);
```

Comment:
You could use the constant here so the comment is not necessary anymore.

```
Redis::instance()->hset(self::redisKey($JobId), 'status', self::STATUS_CANCELLED);
```

Reply:
Done